cluster/addons/rbac/cluster-loadbalancing/glbc/roles.yaml

```diff
@@ -44,12 +44,16 @@ rules:
 - apiGroups: ["extensions", "networking.k8s.io"]
   resources: ["ingresses/status"]
   verbs: ["update", "patch"]
-# GLBC ensures that the `cloud.google.com/backendconfigs` CRD exists in a desired state:
-# https://github.com/kubernetes/ingress-gce/blob/4918eb2f0f484f09ac9e5a975907a9b16ed2b344/cmd/glbc/main.go#L93
+# GLBC ensures that the `cloud.google.com/backendconfigs` and `networking.gke.io/servicenetworkendpointgroups` CRD exists in a desired state:
+# https://github.com/kubernetes/ingress-gce/blob/5c3fcb5845e74b92ea8bd52929b15fc5c9fa7970/cmd/glbc/main.go#L108
+# https://github.com/kubernetes/ingress-gce/blob/5c3fcb5845e74b92ea8bd52929b15fc5c9fa7970/cmd/glbc/main.go#L133
 # TODO(rramkumar1): https://github.com/kubernetes/ingress-gce/issues/744
 - apiGroups: ["apiextensions.k8s.io"]
   resources: ["customresourcedefinitions"]
   verbs: ["get", "list", "watch", "update", "create", "patch"]
 - apiGroups: ["cloud.google.com"]
   resources: ["backendconfigs"]
   verbs: ["get", "list", "watch", "update", "create", "patch"]
+- apiGroups: ["networking.gke.io"]
```

Comment:
I don't have a better suggestion but having a gke specific api group in a generic (cloud provider wise) role file doesn't seem correct.

Reply:
is this actually generic? it's in support of https://github.com/kubernetes/kubernetes/blob/master/cluster/gce/manifests/glbc.manifest, right?

Reply:
So anything gce specific should be under cluster/gce. (Eg. cluster/gce/addons) This is under cluster/addons which *should* be generic. At this point I think this file will only effect gce, e2e and scalability (which is really gce).  So its not effecting much beyond gce.